cmake/OpenCVCompilerOptions.cmake

```diff
@@ -426,6 +426,8 @@ if(MSVC)
     ocv_warnings_disable(CMAKE_CXX_FLAGS /wd4275) # non dll-interface class 'std::exception' used as base for dll-interface class 'cv::Exception'
     ocv_warnings_disable(CMAKE_CXX_FLAGS /wd4512) # Assignment operator could not be generated
     ocv_warnings_disable(CMAKE_CXX_FLAGS /wd4589) # Constructor of abstract class 'cv::ORB' ignores initializer for virtual base class 'cv::Algorithm'
+    ocv_warnings_disable(CMAKE_CXX_FLAGS /wd4819) # Symbols like delta or epsilon cannot be represented
+    ocv_warnings_disable(CMAKE_CXX_FLAGS /wd4702) # unreachable code using Ninja
```

Comment:
> unreachable code

We definitely should not disable such warning globally.

Local `#pragma` should be used instead.

Reply:
Removed this line from [cmake/OpenCVCompilerOptions.cmake](https://github.com/opencv/opencv/pull/22285/files/3e12ebb7aa335c3ce686ab742681f124c7ebb631#diff-1cfc4f7f7baf9f200210e211eedcc4a44a494a3bc3ef887f328aac2f0716b39b) and added `#pragma` to certain files.